rclcpp/include/rclcpp/node.hpp

```diff
@@ -305,6 +311,24 @@ class Node : public std::enable_shared_from_this<Node>
   bool
   get_parameter(const std::string & name, ParameterT & parameter) const;
 
+  /// Assign the value of the map parameter if set into the values argument.
+  /**
+   * Parameter names that are part of a map are of the form "name.member".
+   * This API gets all parameters that begin with name, storing them into the
+   * map with a key of "member" and their value.  If there are no members in
```

Comment:
One sentence per line please.

Reply:
Also, `with a key of "member"` is confusing to me, is the key literally "member" or the name of the parameter?

Reply:
I fixed one sentence per line in f039865cb0cf3b555f7aeefe15f85b4c2af67b12

Also in that commit, I changed the wording here to use "name of the parameter" to reduced confusion.